pkg/target/target_integration_test.go

```diff
@@ -283,7 +283,7 @@ func TestConstraintEnforcement(t *testing.T) {
 				},
```

Comment:
We should modify this file so that we have

* A second function that runs a review against an augmentedUnstructured in addition to the existing AdmissionRequest form
* A common list of tests shared across the two functions

This should not require writing any new test cases, just refactoring the existing ones and creating a second function with a for-loop for going over those test cases.

Reply:
Done -- thanks!